spec/swift/app_spec.rb

```diff
@@ -473,13 +506,23 @@ def app
           get "phil/"
 
           last_response.status.must_equal 200
-          last_response.content_type.must_equal "application/json"
+          last_response.content_type.must_equal "application/ld+json"
 
           content = JSON.parse(last_response.body)
           content["items"]["food/"].wont_be_nil
           content["items"]["food/"]["ETag"].must_equal "f9f85fbf5aa1fa378fd79ac8aa0a457d"
         end
 
+        it "responds with 200 and empty object when directory doesn't exist" do
```

Comment:
It's a non-empty object, but with an empty `items` list/key/property. So I guess returns an "empty directory listing" could work. Or the long version. :)